src/components/datatablesContainer/DataTableSystems/DataTableSystems.module.scss

```diff
@@ -0,0 +1,3 @@
+.returnLink {
```

Comment:
pls change file name to match component file name
class selector has to be wrapped with a namespace so that this style is only applicable to the associated component. See src\components\DataTableRender\DataTableRender.scss

Reply:
Changed file name